kubedifflib/_kube.py

```diff
@@ -46,6 +46,15 @@ def from_dict(cls, data, namespace=""):
         for kube_obj in KubeObject.from_dict(obj, namespace=namespace):
           yield kube_obj
     else:
+      # Transform from e.g. "apps/v1" to what kubectl expects, e.g. "deployment.v1.apps"
+      # From https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get:
+      # "To use a specific API version, fully-qualify the resource,
+      # version, and group (for example: 'jobs.v1.batch/myjob')."
+      api_version_parts = data["apiVersion"].split("/")
```

Comment:
you were looking for a more functional way of doing this, maybe:

```
>>> ".".join(reversed("apps/v1".split("/")))
'v1.apps'
>>> ".".join(reversed("v1".split("/")))
'v1'
```

Reply:
we don't want `v1` we want `v1.`

Reply:
we do want `v1.`, checked loally as well.